src/acl.c

```diff
@@ -1524,6 +1524,16 @@ static int ACLSelectorCheckKey(aclSelector *selector, const char *key, int keyle
     return ACL_DENIED_KEY;
 }
 
+/* Returns if a given command may possibly access channels. For this context,
+ * the unsubscribe commands do not have channels. */
+static int ACLDoesCommandHaveChannels(struct redisCommand *cmd) {
+    return (cmd->proc == publishCommand
+        || cmd->proc == subscribeCommand
+        || cmd->proc == psubscribeCommand
+        || cmd->proc == spublishCommand
+        || cmd->proc == ssubscribeCommand);
```

Comment:
```suggestion
return (cmd->proc == publishCommand ||
    cmd->proc == subscribeCommand ||
    cmd->proc == psubscribeCommand ||
    cmd->proc == spublishCommand ||
    cmd->proc == ssubscribeCommand);
```

Reply:
I prefer putting the boolean operator on the newline so it's clearer what operation is being done. I see it inconsistently done in Redis.